addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -225,69 +225,50 @@ options.registry.carousel = options.Class.extend({
      * @override
      */
     start: function () {
-        var self = this;
-
-        this.$target.carousel({interval: false});
-        this.id = this.$target.attr('id');
-        this.$inner = this.$target.find('.carousel-inner');
-        this.$indicators = this.$target.find('.carousel-indicators');
         this.$target.carousel('pause');
-        this._rebindEvents();
-
-        var def = this._super.apply(this, arguments);
-
-        // set background and prepare to clean for save
-        this.$target.on('slid.bs.carousel', function () {
-            self.$target.carousel('pause');
-            self.trigger_up('option_update', {
-                optionNames: ['background', 'background_position', 'colorpicker', 'sizing_y'],
-                name: 'target',
-                data: self.$target.find('.carousel-item.active'),
-            });
+        this.$indicators = this.$target.find('.carousel-indicators');
+        this.$controls = this.$target.find('.carousel-control-prev, .carousel-control-next, .carousel-indicators');
+        this.$items = this.$target.find('.carousel-item');
+        this.$target.on('slid.bs.carousel.carousel_option', () => {
+            this._retargetEdition();
+            // FIXME why is this needed dammit [sad] => doesn't seem to happen in Firefox
```

Comment:
@sdegueldre I am using firefox... strange if you don't have the bug ;)

Reply:
@sdegueldre Actually, did you find the perfect solution for this ? Still working on your side with a setTimeout 0 ?

Reply:
Still working with a 0 timeout. But if it wasn't working for you, it's probably not reliable. As I was saying, since it's caused by the scroll-bar appearing, adding `max-width: 100%; overflow-x: hidden;` on the body also solves the problem, but doesn't work on the `#wrapwrap`, suggesting it's not the carousel, but the manipulators, especially since the scroll-bar doesn't appear at all when manipulators are `d-none`. But while we probably don't ever want the body to overflow horizontally, adding an `overflow-x: hidden` on the body seems like the kind of change that's susceptible to cause headaches down the road.

I'm thinking that maybe using `window.requestAnimationFrame()` instead of a 0 timeout might be more reliable.